### PR TITLE
V12 patch

### PR DESCRIPTION
**Pull request by apaparazzi0329**
Version 12 patch in response to https://github.com/watson-developer-cloud/java-sdk/issues/1226

### CODE REVIEW

File: assistant/src/main/java/com/ibm/watson/assistant/v1/model/DialogNodeOutputGenericDialogNodeOutputResponseTypeConnectToAgent.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 public class DialogNodeOutputGenericDialogNodeOutputResponseTypeConnectToAgent
     extends DialogNodeOutputGeneric {
 
-@SerializedName("transfer_info")
+  @SerializedName("transfer_info")
```
Comment (jeff-arn) [inline assistant/src/main/java/com/ibm/watson/assistant/v1/model/DialogNodeOutputGenericDialogNodeOutputResponseTypeConnectToAgent.java:23]: Formatting change?

File: speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/model/RecognizeWithWebsocketsOptions.java [hunk 2]
```diff
@@ -218,6 +225,7 @@ public static class Builder {
     private Boolean timestamps;
     private Boolean profanityFilter;
     private Boolean smartFormatting;
+    private Long smartFormattingVersion;
```
Comment (jeff-arn) [inline speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/model/RecognizeWithWebsocketsOptions.java:228]: This is a `Long`?

**Review by jeff-arn — COMMENTED** [review]
Two questions

File: assistant/src/main/java/com/ibm/watson/assistant/v1/model/DialogNodeOutputGenericDialogNodeOutputResponseTypeConnectToAgent.java (hunk 1 above)
Comment (apaparazzi0329) [reply assistant/src/main/java/com/ibm/watson/assistant/v1/model/DialogNodeOutputGenericDialogNodeOutputResponseTypeConnectToAgent.java:23]: Yes, not sure why the formatting was wrong initially but the spacing looks correct here

File: speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/model/RecognizeWithWebsocketsOptions.java (hunk 2 above)
Comment (apaparazzi0329) [reply speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/model/RecognizeWithWebsocketsOptions.java:228]: Yes this is how other "integer" types are generated in the java sdk

Comment (apaparazzi0329) [reply speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/model/RecognizeWithWebsocketsOptions.java:228]: Or to be more specific, how params given the type "integer" in the api definitions are generated for the java sdk

**Review by jeff-arn — APPROVED** [review]

**Pull request merged**